Cargo.toml

```diff
@@ -15,6 +15,7 @@ env_logger = { version = "0.9", default-features = false }
 log = "0.4"
 object = { version = "0.26", default-features = false, features = ["read_core", "elf", "std"] }
 getrandom = "0.2"
+regex = "1.5"
```

Comment:
Why are you adding `regex` anyways? It does not seem to be used so far, or is it?

Reply:
The reason is that it is used in [the next PR](https://github.com/knurling-rs/flip-link/pull/69) and I cherry picked it. But at this stage it is useless. Let me remove it.